main.go

```diff
@@ -842,21 +842,32 @@ func getDefaultPort(portFlag string, usbInterfaces []string) (port string, err e
 }
 
 func usage() {
+	version := goenv.Version
+	if strings.HasSuffix(goenv.Version, "-dev") && gitSha1 != "" {
+		version += "-" + gitSha1
+	}
+
 	fmt.Fprintln(os.Stderr, "TinyGo is a Go compiler for small places.")
-	fmt.Fprintln(os.Stderr, "version:", goenv.Version)
+	fmt.Fprintln(os.Stderr, "version:", version)
 	fmt.Fprintf(os.Stderr, "usage: %s command [-printir] [-target=<target>] -o <output> <input>\n", os.Args[0])
```

Comment:
Shouldn't the `-printir` here be changed to flags?
It might make more sense to surround the command and possibly flags with <> so it's clear this is part of the input.

Reply:
We might as well just write `usage: go <command> [arguments]` like Go.
The help for the part you pointed out probably hasn't been updated for a long time.


```
$ go
Go is a tool for managing Go source code.

Usage:

        go <command> [arguments]

The commands are:

        bug         start a bug report
        build       compile packages and dependencies
        clean       remove object files and cached files
...
```

Reply:
Yeah, good idea!

Reply:
Actually, before that we should probably implement help for each command, otherwise is hard to know which flags/arguments apply to what commands, I reckon.

Reply:
The help for each command should be more extensive.
I think it should be separate from this PR, and aimed at the next version after 0.19.0.
I hope someone will make it.

The most detailed help in existence is the following.
https://tinygo.org/docs/reference/usage/

Reply:
Improved slightly with ad2bb7dd3d46b5287e268a34cc46ab19ab0daa36

new:

```
$ tinygo
No command-line arguments supplied.
TinyGo is a Go compiler for small places.
version: 0.19.0-dev-ad2bb7dd
usage: C:\tinygo\tinygo\build\tinygo.exe <command> [arguments]

commands:
  build:   compile packages and dependencies
  run:     compile and run immediately
  test:    test packages
  flash:   compile and flash to the device
  gdb:     run/flash and immediately enter GDB
  env:     list environment variables used during build
  list:    run go list using the TinyGo root
  clean:   empty cache directory (C:\Users\sago3\AppData\Local\tinygo)
  targets: list targets
  info:    show info for specified target
  version: show version
  help:    print this help text

for more details, see https://tinygo.org/docs/reference/usage/
```